Netconf-notif-v12-examples-v17Sept2018/netconf-notif-11-subscription-started.xml

```diff
@@ -2,13 +2,13 @@
   <eventTime>2007-09-01T10:00:00Z</eventTime>
   <subscription-started 
       xmlns="urn:ietf:params:xml:ns:yang:ietf-subscribed-notifications"
-	  xmlns:nsn=
-    "urn:ietf:params:xml:ns:yang:ietf-netconf-subscribed-notifications">
+      xmlns:nsn="urn:ietf:params:xml:ns:yang:ietf-foo-subscribed-notifications">

```

Comment:
Example may be removed for this iteration, but when re-instated changes here will be required.